modules/vcf2maf/1.0/vcf2maf.smk

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Author:  Bruno Grande
+# Module Author:    Helena Winata
+# Contributors:     N/A
+
+
+##### SETUP #####
+
+import sys, os
+from os.path import join
+
+import oncopipe as op
+
+# Setup module and store module-specific configuration in `CONFIG`
+CONFIG = config["lcr-modules"]["vcf2maf"]
+LOG = "/logs/" + op._session.launched_fmt
+
+wildcard_constraints:
+    prefix = "[0-9]{2}-.*"
+
+VERSION_UPPER = {
+    "grch37": "GRCh37",
+    "GRCh37": "GRCh37",
+    "grch38": "GRCh38",
+    "GRCh38": "GRCh38",
+}
+
+##### RULES #####
+
+rule _vcf2maf_run:
+    input:
+        vcf = "{out_dir}/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{vcf_name}.vcf",
+        fasta = reference_files("genomes/{genome_build}/genome_fasta/genome.fa"),
+        vep_cache = CONFIG["inputs"]["vep_cache"]
```

Comment:
It looks like this assumes the user installed the cache themselves. Why did we do it this way instead of using my auto-install approach?

Reply:
I haven't implemented the auto-install approach yet since I wasn;t sure if it should be part of the `reference_files` subworkflow instead